app/src/main/res/layout/item_checkbox_string_description.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/parent_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingLeft="@dimen/list_two_line_padding_vertical"
+    android:paddingRight="@dimen/list_two_line_padding_vertical"
+    android:paddingTop="@dimen/list_two_line_padding_horizontal"
+    android:paddingBottom="@dimen/list_two_line_padding_horizontal"
+    android:orientation="horizontal"
+    android:gravity="center_vertical"
+    android:background="?android:attr/selectableItemBackground"
+    android:clickable="true">
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingLeft="@dimen/list_two_line_padding_vertical"
```

Comment:
The padding here should be removed, since it is applied twice (here and the parent LinearLayout)